test/controllers/profiles_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class ProfilesControllerTest < ActionController::TestCase
+  test "should get create" do
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.